lib/inputs/rpm/static.ts

```diff
@@ -38,3 +45,42 @@ export async function getRpmDbFileContent(
     return "";
   }
 }
+
+export async function getRpmSqliteDbFileContent(
+  extractedLayers: ExtractedLayers,
+): Promise<PackageInfo[]> {
+  const rpmDb = getContentAsBuffer(
+    extractedLayers,
+    getRpmSqliteDbFileContentAction,
+  );
+  if (!rpmDb) {
+    return [];
+  }
+
+  try {
+    const fileObj = tmp.fileSync();
+    await writeFile(fileObj.fd, rpmDb);
+
+    const results: IParserSqliteResponse = await getPackagesSqlite(
+      fileObj.name,
+    );
+
+    fileObj.removeCallback(); // removing the temp file created after processing
+    return results.response;
```

Comment:
if we only use `response` out of results, why do we return an object and not whatever's in the response?

Reply:
Sorry this was outdated, handling the error as well